db/patterns/adobe_experience_cloud.eno

```diff
@@ -14,6 +14,7 @@ sitestat.com
 --- domains
 
 --- filters
+|https://smetrics\.(.+)\?
```

Comment:
```suggestion
|https://smetrics.
```

Should work. It matches all URLs that start with "https://smetrics."

(Note that "|text" is a normal string comparison. So, quoting like for regular expressions will not have the expected effect.)

Reply:
Makes sense. Thanks for clarifying how the "starts with" filter works.